generate.go

```diff
@@ -127,6 +127,13 @@ func cloneTemplate(pdk pdkTemplate, dir, tag string) error {
 		return err
 	}
 
+	// initialize submodules if any
```

Comment:
Did you explore just adding `--recursive` to the original clone command?

Reply:
this makes things much nicer!